### PR TITLE
DOC: Update documentation folder

### PR DESCRIPTION
**Pull request by lboucin**

### CODE REVIEW

File: doc/source/api/index.rst [hunk 1]
```diff
@@ -4,33 +4,34 @@
 API reference
 =============
 
-This section describes the core Pythonic interfaces for Twin Runtimes.
-Here, you can find the application programming interfaces for 
-using the core Twin Runtime SDK functionalities, as well as an example
-of higher level abstraction class to evaluate the Runtimes.
+This section describes the core Pythonic interfaces for twin runtimes.
+Here, you can find the APIs for using twin runtime functionalities and
+the higher-level abstraction class for evaluating twin runtimes.
```
Comment (PipKat) [inline doc/source/api/index.rst:9]: @lboucin I am not sure I understand the distinction made in lines 8 and 9 between using twin runtime functionalities and evaluating twin runtimes. Is there a better way to explain what is intended here?

**Review by PipKat — APPROVED** [review]

File: doc/source/api/index.rst (hunk 1 above)
Comment (lboucin) [reply doc/source/api/index.rst:9]: @PipKat We want to emphasize the distinction between the TwinRuntime and TwinModel APIs. Would following variation make more sense? "Here, you can find all APIs for consumming twin runtimes, from the lowest-level APIs with the :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class to the higher-level with the :class:`TwinModel <ansys.pytwin.TwinModel` class."

File: doc/source/api/index.rst [hunk 2]
```diff
@@ -4,33 +4,34 @@
...
 
-Twin Runtime SDK
-----------------
+Twin runtimes
+-------------
 
-Implementation of the :ref:`ref_index_api_sdk` class enabling access to the core
-Twin Runtime SDK functionalities.
+The :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class provides access to
+twin runtime functionalities. For a workflow example, see :ref:`ref_index_api_sdk`.
```
Comment (lboucin) [inline doc/source/api/index.rst:15]: @PipKat. Following your comment, I think we can better emphasize here the distinction. I suggest:
"The :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class provides access to all the twin runtime functionalities. It is the lowest level API of the twin runtime SDK. For a workflow example, see ..."
What do you think? Thanks

File: doc/source/api/index.rst (hunk 1 above)
Comment (PipKat) [inline doc/source/api/index.rst:9]:
```suggestion
Here, you can find all APIs for consuming twin runtimes, from the lowest-level APIs
with the :class:TwinRuntime <ansys.pytwin.TwinRuntime class to the higher-level
APIs with the :class:TwinModel <ansys.pytwin.TwinModel class.
```

Comment (PipKat) [reply doc/source/api/index.rst:9]: @lboucin I like this much better and changed the file! Thanks!

File: doc/source/api/index.rst (hunk 2 above)
Comment (PipKat) [inline doc/source/api/index.rst:15]:
```suggestion
The :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class provides access to all
the twin runtime functionalities. It is the lowest-level API of the twin runtime SDK.
For a workflow example, see :ref:`ref_index_api_sdk`.
```

Comment (PipKat) [reply doc/source/api/index.rst:15]: @lboucin Perfect! Change made and incorporated.

**Review by chrpetre — APPROVED** [review]
LGTM, thanks!

**Pull request merged**